test/remote/check_tests.js

```diff
@@ -28,9 +28,15 @@ test(
 
 var client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
-});
+})
+
+var ACTIONS = [
+  'accountCreate',
+  'accountLogin',
+  'passwordChange'
+]
 
-['accountCreate', 'accountLogin', 'passwordChange'].forEach(function (action) {
+ACTIONS.forEach(function (action) {
```

Comment:
I think this is an ASI issue. Since the preceding expression ends with `)`, it's collapsing to `restify.createJsonClient({ ... })['accountCreate', ...]`.

Does it still complain if you write `;['accountCreate', 'accountLogin', 'passwordChange'].forEach(...)`?